test/fixtures/parser/mysql/call_statement.sql

```diff
@@ -1 +1 @@
-CALL zproc_validate_send_adminuser_otp('a');
\ No newline at end of file
+CALL somefunction('a');
```

Comment:
Is there a reason for changing the function name in this and other test `.sql` files?

Reply:
I used a name of a proc that is specific to me and wanted to make it generic instead.